### DISCUSSION TITLE
Another KVM vendor tested

### DISCUSSION THREAD

**Discussion by coryhamma** [Ysurac/openmptcprouter#1568] [Show and tell]
I just tested BuyVM.net which offers KVM services. I used their "Slice 1024" which gives you 1 GB RAM and unlimited bandwidth for $3.50 per month. You can also purchase additional ipv4 addresses at $2/month.

It seems to work great. I had one small problem where the IP address they gave me was already geotagged as North Korea! This was fun until my wife asked me why all her Google search results are returning in Korean. Once I asked them to give me a different IP address, I had no problems. 

Using this service, I'm able to get 45-60 mbps with my combined connections. I average 60-120 mbps on all the connections separately.

**Comment by Magnum1981** [comment]
Kindly explain this. I'm new to openmptcp and from what I have read our combined speed using openmptcp should be better but it looks like your combined speed reduced from using it. Thank you for taking time to explain
